frontend/public/components/pod.tsx

```diff
@@ -104,140 +109,209 @@ const tableColumnClasses = [
+  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
+
   Kebab.columnClass,
 ];
 
 const kind = 'Pod';
+const columnManagementID = referenceForModel(PodModel);
 
 const podRowStateToProps = ({ UI }) => ({
   metrics: UI.getIn(['metrics', 'pod']),
+  selectedColumns: UI.getIn(['columnManagement']),
 });
 
-const PodTableRow = connect<PodTableRowPropsFromState, null, PodTableRowProps>(podRowStateToProps)(
-  ({
-    obj: pod,
-    index,
-    rowKey,
-    style,
-    metrics,
-    showNodes,
-  }: PodTableRowProps & PodTableRowPropsFromState) => {
-    const { name, namespace, creationTimestamp } = pod.metadata;
-    const { readyCount, totalContainers } = podReadiness(pod);
-    const phase = podPhase(pod);
-    const restarts = podRestarts(pod);
-    const bytes: number = _.get(metrics, ['memory', namespace, name]);
-    const cores: number = _.get(metrics, ['cpu', namespace, name]);
-    return (
-      <TableRow id={pod.metadata.uid} index={index} trKey={rowKey} style={style}>
-        <TableData className={tableColumnClasses[0]}>
-          <ResourceLink kind={kind} name={name} namespace={namespace} />
-        </TableData>
-        <TableData className={classNames(tableColumnClasses[1], 'co-break-word')}>
-          <ResourceLink kind="Namespace" name={namespace} />
-        </TableData>
-        <TableData className={tableColumnClasses[2]}>
-          <Status status={phase} />
-        </TableData>
-        <TableData className={tableColumnClasses[3]}>
-          {readyCount}/{totalContainers}
-        </TableData>
-        <TableData className={tableColumnClasses[4]}>{restarts}</TableData>
-        <TableData className={tableColumnClasses[5]}>
-          {showNodes ? (
-            <ResourceLink kind="Node" name={pod.spec.nodeName} namespace={namespace} />
-          ) : (
-            <OwnerReferences resource={pod} />
-          )}
-        </TableData>
-        <TableData className={tableColumnClasses[6]}>
-          {bytes ? `${formatBytesAsMiB(bytes)} MiB` : '-'}
-        </TableData>
-        <TableData className={tableColumnClasses[7]}>
-          {cores ? `${formatCores(cores)} cores` : '-'}
-        </TableData>
-        <TableData className={tableColumnClasses[8]}>
-          <Timestamp timestamp={creationTimestamp} />
-        </TableData>
-        <TableData className={tableColumnClasses[9]}>
-          <ResourceKebab
-            actions={menuActions}
-            kind={kind}
-            resource={pod}
-            isDisabled={phase === 'Terminating'}
-          />
-        </TableData>
-      </TableRow>
-    );
-  },
-);
-PodTableRow.displayName = 'PodTableRow';
-
 const getHeader = (showNodes) => {
   return () => {
     return [
       {
         title: 'Name',
+        ID: 'NAME',
         sortField: 'metadata.name',
         transforms: [sortable],
         props: { className: tableColumnClasses[0] },
+        visible: true,
       },
       {
         title: 'Namespace',
+        ID: 'NAMESPACE',
```

Comment:
Prefer lowercase to all caps

```suggestion
        id: 'namespace',
```

Reply:
done